distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -2725,6 +2726,8 @@ type DB_Table
                         warnings_builder.append (Inexact_Type_Coercion.Warning expected_type actual_type)
 
             result = max_rows.attach_warning materialized_table
+            result.if_not_error <|
+                Telemetry.log "DB_Table.read" "Materialize Table: connection_type={}, output={}, row_count={}, column_count={}" [self.connection.to Meta.Type . name, "Table", result.row_count, result.column_count]
```

Comment:
Is it worth also knowing if `max_rows` was set to some limit or unlimited? This way we know if the full table or only a sample was read.

Not sure how useful is that.